version: "3"
services:
  plex:
    image: plexinc/pms-docker:1.29.0.6244-819d3678c
    restart: unless-stopped
    environment:
      - TZ=Europe/Stockholm
      - PLEX_CLAIM=${PLEX_CLAIM}
      - PUID=1000
      - PGID=1000
      - VERSION=docker
    networks:
      - traefik
    volumes:
      - ${PLEX_DATA}/config:/config
      - ${PLEX_DATA}/data:/data
    labels:
      - "traefik.enable=true"
      - "traefik.http.routers.plex.rule=Host(`${PLEX_HOST}`)"
      - "traefik.http.services.plex.loadbalancer.server.port=32400"

networks:
  traefik:
    external: true
